types/src/main.ts

```diff
@@ -669,6 +669,61 @@ export namespace TextDocumentItem {
 	}
 }
 
+/**
+ * Describes the content type that a client supports in various
+ * result literals like `Hover`, `ParameterInfo` or `CompletionItem`
+ */
+export namespace MarkupKind {
+	/**
+	 * Markdown is supported as a content format
+	 */
+	export const Markdown: 1 = 1;
```

Comment:
Maybe better a string so this can be extended without the need of change in our code.

Reply:
I would add PlainText as well so 'MarkdownContent' is the only needed type in representing formatted strings.
Not really needed for the completion documentation (there you can pass plain text as string), but for Hover there currently no way of just passing a plain text (it has to be MarkedString, and plain text needs to be escaped).
Especially good to have for future types with a documentation field.